.github/workflows/nf-core-linting.yml

```diff
@@ -0,0 +1,71 @@
+name: nf-core linting
+# This workflow is triggered on pushes and PRs to the repository.
+# It runs the `nf-core lint` tests to ensure that the module code meets the nf-core guidelines
+on: [push, pull_request]
+
+jobs:
+  changes:
+    name: Check for changes
+    runs-on: ubuntu-latest
+    outputs:
+      # Expose matched filters as job 'modules' output variable
+      modules: ${{ steps.filter.outputs.changes }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: 'tests/config/pytest_software.yml'
+
+  test:
+    runs-on: ubuntu-20.04
+    name: ${{ matrix.tags }} ${{ matrix.nxf_version }}
+    needs: changes
+    if: needs.changes.outputs.modules != '[]'
+    strategy:
+      fail-fast: false
+      matrix:
+        nxf_version: ['21.03.0-edge']
+        tags: ['${{ fromJson(needs.changes.outputs.modules) }}']
+    env:
+      NXF_ANSI_LOG: false
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
```

Comment:
You're trying to load with key `${{ runner.os }}-pip-` but saving the cache with key `${{ runner.os }}-nextflow-`

Reply:
No idea what this is doing to be honest.